package.json

```diff
@@ -434,6 +434,12 @@
                 "category": "PostgreSQL",
                 "command": "postgreSQL.openStoredProcedure",
                 "title": "Open Stored Procedure"
+            },
+            {
```

Comment:
This is what it looks like
<img width="397" alt="image" src="https://github.com/microsoft/vscode-cosmosdb/assets/39359541/10583a9d-fa12-4942-a0f9-91632f22ec81">